### PR TITLE
[MSKINS-242] Replace deprecated prettify JS library

### PR DESCRIPTION
**Pull request by michael-o**
@kwin Here is a very early draft. Problematic. Bootstrap's padding and highlight.js padding add up and make it abysmal:
grafik

It will take time to make it look like:
grafik

### CODE REVIEW

**Comment by michael-o** [comment]
This draft includes these languages:
"bash.js"
"bash.min.js"
"diff.js"
"diff.min.js"
"java.js"
"java.min.js"
"json.js"
"json.min.js"
"powershell.js"
"powershell.min.js"
"shell.js"
"shell.min.js"
"xml.js"
"xml.min.js"

File: pom.xml [hunk 1]
```diff
@@ -136,11 +137,13 @@ under the License.
         <!-- add skin-info -->
       </resource>
 
-      <!-- include the print.css -->
+      <!-- include the some files -->
```
Comment (kwin) [inline pom.xml:140]: The comment is not helpful and contains a typo, let us remove it altogether.

File: src/main/resources/META-INF/maven/site-macros.vm [hunk 2]
```diff
@@ -48,7 +48,7 @@
 #**##end
 #end
 ##
-#macro( link $link $icon = false $omitImage = false $elemOnEmptyHref = "a" $dotSlashOnEmptyHref = false $imageClass = "imageLink" $overrideElemClass = false )
+#macro( link $link $iconClass = null $omitImage = false $elemOnEmptyHref = "a" $dotSlashOnEmptyHref = null $imageClass = "imageLink" $overrideElemClass = null )
```
Comment (kwin) [inline src/main/resources/META-INF/maven/site-macros.vm:51]: How did these changes slip into this PR? They are not related.

Comment (michael-o) [reply src/main/resources/META-INF/maven/site-macros.vm:51]: This looks like a bug in GH, ignore it.

**Comment by michael-o** [comment]
@kwin, do you want to work on this? Before 2.0.p

**Comment by jira-importer** [comment]
Resolve #277

**Pull request closed without merging**